src/hockeypuck/go.mod

```diff
@@ -19,13 +19,15 @@ require (
 	github.com/goods/httpbuf v0.0.0-20120503183857-5709e9bb814c // indirect
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/interpose/middleware v0.0.0-20150216143757-05ed56ed52fa // indirect
+	github.com/jmcvetta/randutil v0.0.0-20150817122601-2bb1b664bcff
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/justinas/nosurf v0.0.0-20190416172904-05988550ea18 // indirect
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
 	github.com/kr/pretty v0.2.1 // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/lib/pq v1.8.0
 	github.com/meatballhat/negroni-logrus v0.0.0-20170801195057-31067281800f // indirect
+	github.com/mroth/weightedrand v0.4.1
```

Comment:
This doesn't seem to be imported. Is this leftover from evaluating weightedrand?

Running `go mod tidy` should clean this up.

Reply:
Yes, indeed. Until I noticed that `randutil.WeightedChoice` would solve it in one go.